.ci/travis/checkstyle_regression_no_exception.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+BRANCH_COMMIT="$1"
+MASTER_COMMIT="$2"
+CHECKSTYLE_VERSION="$3"
+
+if [ ! -d checkstyle ]; then
+    git clone https://github.com/checkstyle/checkstyle
+fi
+
+if [ ! -d contribution ]; then
+    git clone --depth=1 https://github.com/checkstyle/contribution
```

Comment:
At the beginning I wrote like this as well, but found some problems and changed it to delete the repo every time. But I can't remember the reason.
Now there is only one test with specific commit id. Maybe there would be problems when we test twice(HEAD and specific)